### ISSUE TITLE
javascrip prompt() function not working

### ISSUE THREAD

**Issue by jheroy** [Jelmerro/Vieb#369]
**Describe the bug**
Code: 
```
var name=prompt("Please input your name:", "");
if ( name == "") {
    alert("Invalid name!")
}
```
Vieb will  directly popup alert  `Invalid name!` instead prompt a question window.

**Specs**
 - OS: macOS monterey
 - Version: 7.2.0

**Comment by Jelmerro** [comment]
Prompts are not supported by Electron [as explained here](https://github.com/electron/electron/issues/472), but when reading through the issue again I found [this library](https://github.com/ssight/native-prompt) to maybe add it.

Please do note that you can already be notified of when a page tries to use prompts by running the:
`:set dialogprompt=notifyblock` command, similar to `dialogalert` and `dialogconfirm`. However, unlike these last two, the prompt can't currently be allowed as it's not implemented. I'll see if the linked library might be of help to solve this, but otherwise the `dialogprompt` setting will remain your only option to at least know that the page tried to use a prompt. The default response is to cancel the prompt (and other dialogs as well).

**Comment by Jelmerro** [comment]
After some short experimentation, I can conclude that this can definitely be made working with either a plugin or just a custom browserwindow that's displayed in sync and passes back the data and closes when pressing enter. I will work on this soon.

**Comment by jheroy** [comment]
Nice.

**Comment by Jelmerro** [comment]
Implemented in [the new 8.0.0 release](https://github.com/Jelmerro/Vieb/releases/tag/8.0.0), keep in mind that all dialogs are blocked by default, but they can all be enabled including prompt! See `help dialogprompt` for info, but in short, if you want prompts to work, run `:set dialogprompt=show`.
